### DISCUSSION TITLE
How to add cookie notice?

### DISCUSSION THREAD

**Discussion by normanwankowski** [mozilla/hubs#4725] [Hubs Cloud Questions]
It is a legal requirement to add a cookie consent since I am using user data and Google Analytics. How can I add my 3rd party custom embed code in order to show my cookie pop-up ?

**Comment by pattersonbl2** [comment]
I would consult with a your legal team about the cookie consent.  We currently don't have a standard way to added the cookie pop up I would reach out to the community discord to see if someone has been able to create one.

**Reply by normanwankowski** [reply]
My plan was to use https://cookie-script.com they offer a free script for the cookie banner. So my question is more where can I add it so that it is visible on my hubs homepage? Is there a away to add custom html scripts to the header?

**Reply by pattersonbl2** [reply]
You would have to go through the process of editing the frontend code to add a script for cookies. I would ask this question in the community discord to see if others have achieved this because the process may be a little complex.

**Reply by normanwankowski** [reply]
In the Hubs backend under Server Settings - Advanced I found the option  EXTRA HOME PAGE SCRIPT and EXTRA HOMEPAGE HEADER HTML but when I add my Cookie Script it says: 

_Refused to execute a script because its hash, its nonce, or 'unsafe-inline' does not appear in the script-src directive of the Content Security Policy._

Is there a way to fix that? I will ask in Discord aswell. Thx anyways. I hope someone else is also facing this problem.

**Reply by pattersonbl2** [reply]
Tested the Homepage header HTML, it gives the ability to add extra wording to the top right.    

Using the Homepage script, I was able to upload a simple js script but it didn't process on the homepage during browser refresh

Hopefully someone in discord has already found a fix for this. I will keep doing some troubleshooting on my end
